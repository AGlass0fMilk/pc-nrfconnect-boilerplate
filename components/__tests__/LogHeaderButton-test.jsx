import React from 'react';
import renderer from 'react-test-renderer';
import {
    decorateLogHeaderButton,
} from '../LogHeaderButton';

describe('LogHeaderButton', () => {
    it('should implement component decorator', () => {
        expect(decorateLogHeaderButton).toBeInstanceOf(Function);
    });
});

describe('decorateLogHeaderButton', () => {
    it('should render component', () => {
        const LogHeaderButton = decorateLogHeaderButton('div');
        expect(renderer.create(
            <LogHeaderButton />,
        )).toMatchSnapshot();
    });
});
