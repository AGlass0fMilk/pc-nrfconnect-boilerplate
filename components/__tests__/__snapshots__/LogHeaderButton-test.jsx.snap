exports[`decorateLogHeaderButton should render component 1`] = `<div />`;
